docs/book/misc/debugging.md

```diff
@@ -0,0 +1,234 @@
+---
+description: A guide to debug common issues and get help.
+---
+
+# Debugging Guide
+This page documents a series of how-tos for efficient debugging and getting help without losing your hair.
+
+## 💡 Introduction
+If you stumbled upon this page, chances are you're facing issues with using ZenML. First, thank you for your interest in ZenML!
+We want to make your journey with ZenML as easy and smooth as possible when faced with issues.
+This page documents suggestions and best practices to let you debug, get help, and solve issues quickly.
+
+## 🔍 Getting help
+The most direct and quickest path to get help is on our Slack. 
```

Comment:
I would not encourage people to ask for help on slack. First I would guide them to get additional logs, get server and client logs, and try to understand the problem. Only thne shift them to slack

Reply:
Fixed.